packaging/amazon-linux-ami/agent.service

```diff
@@ -0,0 +1,29 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://github.com/aws/amazon-ecs-agent
+Requires=docker.service
+After=docker.service
+After=cloud-final.service
+[Service]
```

Comment:
nit: add a space between blocks
(Same for each duplicate `agent.service` file below.)

follow the pattern here:
https://github.com/aws/amazon-ecs-init/blob/master/packaging/amazon-linux-ami/ecs.service